server/server.py

```diff
@@ -54,6 +54,28 @@ def admin():
 def connect(sid, environ):
   print("connect ", sid)
 
+@sio.on('cipher_ping')
+def cipherconnect(sid, environ):
+  table = int(CLIENTS[sid])
+  message = 'gateway'
+  cipher = encrypt(message,table)
+  sio.emit('cipher_return', cipher)
+
+
+def encrypt(message, shift):
+  alphabet = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
python has the alphabet in a constant already that we can use - https://docs.python.org/2/library/string.html#string.ascii_lowercase